content/neuvector-prime/azure/contents.lr

```diff
@@ -149,8 +158,73 @@ Billing will be available in Azure Portal billing
 
 Home > Cost Management <subscription> | Cost analysis
 
+## Troubleshooting
+
+This section contains information to help you troubleshoot issues when install Neuvector and configure Billing-adapter
+
+After successful deployment, it should list similar pod and chart output 
+
+```
+kubectl get deployments --all-namespaces=true
+NAMESPACE     NAME                                                      READY   UP-TO-DATE   AVAILABLE   AGE
+kube-system   addon-http-application-routing-external-dns               1/1     1            1           19m
+kube-system   addon-http-application-routing-nginx-ingress-controller   1/1     1            1           19m
+kube-system   coredns                                                   2/2     2            2           19m
+kube-system   coredns-autoscaler                                        1/1     1            1           19m
+kube-system   extension-agent                                           1/1     1            1           16m
+kube-system   extension-operator                                        1/1     1            1           16m
+kube-system   konnectivity-agent                                        2/2     2            2           19m
+kube-system   metrics-server                                            2/2     2            2           19m
+neuvector     neuvector-controller-pod                                  3/3     3            3           15m
+neuvector     neuvector-csp-pod                                         1/1     1            1           15m
+neuvector     neuvector-manager-pod                                     1/1     1            1           15m
+neuvector     neuvector-scanner-pod                                     3/3     3            3           15m
+```
+
+```
+helm list -n neuvector
+NAME         	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART     	APP VERSION
+testextension	neuvector	4       	2023-09-13 15:39:03.266748698 +0000 UTC	deployed	core-2.6.1	5.2.0 
+```
+
+Jobs and Pods
```

Comment:
I assume this is meant as a sub-heading, I wonder if there's a formatting option to distinguish it from the plain text lines